### PR TITLE
Ready to merge: michelle's portfolio

### PR DESCRIPTION
**Pull request by MichelleGolle**
WIP

### CODE REVIEW

File: students/michelle_golle/20150814-michelle_golle-m4.markdown [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Michelle Golle - M4 Portfolio
+
+## Individual
+
+### Areas of Emphasis
+
+Coming into this module I felt weak on JavaScript. After our classes this module along with many Egghead videos/tutorials I feel much more confident.
```
Comment (stevekinney) [inline students/michelle_golle/20150814-michelle_golle-m4.markdown]: :smile:

**Comment by blocknroll** [comment]
It's been really great getting to work with you and getting to know you better during the last two modules, Michelle!  You always bring a nice energy to our group and you've been a real pleasure to work with!

**Comment by tleskin** [comment]
We've gotten to work on two projects together and both times went pretty smoothly. In your portfolio you mention how you've gotten feedback to speak up more and I would agree with that, but I think you're doing awesome at it. Whenever I was stumped in our Game Time project, you seemed to always be able to chime in and help get us going in the right direction, so I would say to just embrace that and do try to speak up more because I think you do have some really valuable ideas.

**Comment by MichelleGolle** [comment]
Ready to merge

**Pull request merged**